CMakeLists.txt

```diff
@@ -60,8 +62,8 @@ find_package(Eigen3 REQUIRED)
 include_directories(${EIGEN3_INCLUDE_DIR})
 
 if (tricktrack_logger)
-  find_package(spdlog REQUIRED)
-  include_directories(${SPDLOG_INCLUDE_DIR})
```

Comment:
Will be `spdlog` always included as part of `tricktrack` or could it be taken as an external package?

Reply:
Could for sure be taken as an external package! I didn't consider this option to be used when installing tricktrack with package manager tools though, is more of an extra for the user / developer.